examples/packet_forward_test.go

```diff
@@ -28,16 +28,19 @@ func TestPacketForwardMiddleware(t *testing.T) {
 
 	cf := ibctest.NewBuiltinChainFactory(zaptest.NewLogger(t), []*ibctest.ChainSpec{
 		{Name: "gaia", ChainName: "gaia-fork", Version: "bugfix-replace_default_transfer_with_router_module"},
-		{Name: "osmosis", ChainName: "osmosis", Version: "v7.3.0"},
-		{Name: "juno", ChainName: "juno", Version: "v6.0.0"},
+		{Name: "osmosis", ChainName: "osmosis", Version: "v11.0.1"},
+		{Name: "juno", ChainName: "juno", Version: "v9.0.0"},
```

Comment:
Thanks for upgrading these. This would be something I'd probably put in a separate PR, though. That way via `git bisect` or similar, if this change were to introduce flaky/failing tests, we can zero in on the cause quicker. 

No changes needed in the PR. Just an FYI.

Reply:
Understood. I updated these while working through fixing the packet forward middleware test for the events processor and decided to keep them in since these versions are isolated to that test.